frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -22,17 +22,21 @@ Feature: Applications managed by GitOps and Create secret
               And user navigates to Environments page
              Then user will see the list GitOps application groupings on the page
 
+        @regression @manual
+        Scenario: Application Details page for Applications: GO-01-TC04
+            Given user is on the Environments page
+              And user can see the Applications on the page
+             Then user can add new environments
```

Comment:
User cannot add new environments directly through console, if that's what this is implying

Reply:
Yes, through its not possible. But using KAM user can add that. Let me rephrase it

Reply:
```
hen user can add new environments using kam cli
```
I think it makes more sense

Reply:
perfect, thank you!